deployer/src/main/java/eu/seaclouds/policy/SeaCloudsInitializerPolicy.java

```diff
@@ -0,0 +1,544 @@
+                            "                  \"legend\": { \"avg\": true, \"current\": false, \"max\": true, \"min\": true, \"show\": true, \"total\": false, \"values\": false}," +
+                            "                  \"lines\": true," +
+                            "                  \"linewidth\": 2," +
+                            "                  \"links\": []," +
+                            "                  \"nullPointMode\": \"connected\"," +
+                            "                  \"percentage\": false," +
+                            "                  \"pointradius\": 5," +
+                            "                  \"points\": false," +
+                            "                  \"renderer\": \"flot\"," +
+                            "                  \"seriesOverrides\": []," +
+                            "                  \"span\": 12," +
+                            "                  \"stack\": false," +
+                            "                  \"steppedLine\": false," +
+                            "                  \"targets\": [{ " +
+                            "                                    \"column\": \"value\"," +
+                            "                                    \"function\": \"mean\"," +
+                            "                                    \"query\": \"select mean(value) from \\\"" + metricName + "\\\" where $timeFilter group by time($interval) order asc\"," +
+                            "                                    \"refId\": \"A\"," +
+                            "                                    \"series\": \"" + metricName + "\"" +
+                            "                               }]," +
+                            "                  \"timeFrom\": null," +
+                            "                  \"timeShift\": null," +
+                            "                  \"title\": \"" + metricName + "\"," +
+                            "                  \"tooltip\": { \"shared\": true, \"value_type\": \"cumulative\"}," +
+                            "                  \"type\": \"graph\"," +
+                            "                  \"x-axis\": true," +
+                            "                  \"y-axis\": true," +
+                            "                  \"y_formats\": [\"short\",\"short\"]" +
+                            "              }]," +
+                            "      \"title\": \"" + entity.getDisplayName() + "\"" +
+                            "},";
+            rowOptions += currentRow;
+        }
+
+        // Remove last ,
+        rowOptions = rowOptions.substring(0, rowOptions.length() - 1);
+        grafanaInitializerPayload = globalGrafanaOptionsJSON + rowOptions + "] } }";
+
+    }
+
+    @Override
+    public void setEntity(EntityLocal entity) {
```

Comment:
Then how we are going to clean up the Agreement, MR and Dashboard? This policy def. needs to do a clean up before it stops.